src/components/Link/linkExtensions.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+/**
+ * A HOC that provides a bit more intelligence / intuitive
+ * function to the React-Router Link
+ */
+
+const isExternal = url => /^(https?:)*\/\//.test(url);
+
+export default WrappedLink => ({ to, children, ...rest }) => (
+  <WrappedLink
+    to={to}
+    {...(isExternal(to) ? { target: '_blank', rel: 'noopener' } : {})}
+    external={isExternal(to)}
```

Comment:
Does this attribute actually do anything? React Router seems to indicate that this will be passed through to the `<a>` tag, but I don't see docs for it in MDN either...

Reply:
It affects the rendering of the styled component. It won't be passed through to the underlying component below that.